src/api/procedures/__tests__/modifyAllowance.ts

```diff
@@ -76,62 +80,36 @@ describe('modifyAllowance procedure', () => {
   });
 
   test('should throw an error if the Subsidy does not exist', async () => {
-    const proc = procedureMockUtils.getInstance<ModifyAllowanceParams, void>(mockContext);
-
-    let error;
-
-    try {
-      await prepareModifyAllowance.call(proc, {
+    await expect(() =>
+      prepareModifyAllowance.call(proc, {
         ...args,
         subsidy: entityMockUtils.getSubsidyInstance({ exists: false }),
-      });
-    } catch (err) {
-      error = err;
-    }
-
-    expect(error.message).toBe('The Subsidy no longer exists');
+      })
```

Comment:
No need for the arrow function, and you can just `return expect` and have the method not be async. Using `await expect` is for when you need to test several things in the same `test` call. That can be changed in multiple places I think